### ISSUE TITLE
Async World Creating / Loading

### ISSUE THREAD

**Issue by xXSchrandXx** [IntellectualSites/PlotSquared#2005]
Hey,
I know you guys already know about FAWE, but can you implement the feature of FAWE with the asyncro world creation or loading? (API: https://github.com/boy0001/FastAsyncWorldedit/wiki/AsyncWorld)

This could help smaller server / performance. It would be nice if you could do that.

With great wishes
xXSchrandXx

**Comment by xXSchrandXx** [comment]
Is there a way to cancel the autoload of the plotworld?
If I load the world Asnyc, I will get the error that the World is Loaded already.

**Comment by RedstoneFuture** [comment]
Yes. If you use multiverse you can use `/mvm set autoload false` in the specifically plotworld.

**Comment by xXSchrandXx** [comment]
I don't use Multiverse, can I deaktivate the autoload / Override from Plotsquared with a specific code?
